e2e/rook-test-framework/contracts/interface_docker_client.go

```diff
@@ -0,0 +1,8 @@
+package contracts
+
+type IDockerClient interface {
```

Comment:
in golang, interfaces aren't supposed to have the `I` prefix or be contained in a filename indicating interface. So you could just call this interface DockerClient